README.md

```diff
@@ -59,6 +59,10 @@ oneMKL is part of [oneAPI](https://oneapi.io).
             <td align="center"><a href="https://github.com/ROCmSoftwarePlatform/rocRAND"> AMD rocRAND</a> for AMD GPU </td>
             <td align="center">AMD GPU</td>
         </tr>
+        <tr>
+            <td align="center"><a href="https://github.com/ROCmSoftwarePlatform/rocFFT">AMD rocFFT</a> for AMD and NVIDIA GPU </td>
```

Comment:
While rocFFT does support running NVIDIA GPUs, we don't provide support for that.

Reply:
My opinion is that mentioning NVidia GPU here is confusing. 

On the whole I'm confused by the way we seem to have *<Libname> for <Targets>*, then in the next column *<Targets>*. Perhaps as a precident, the closed MKL MKLGPU interface does actually support using the CPU if you use a CPU queue, but only mentions GPU. That said, I didn't put it in the table in that PR out of ignorance rather than intentionally.